app/static/js/pico_recipe.js

```diff
@@ -18,6 +18,17 @@ var default_data = [{name:"Preparing To Brew", location:"Prime", temperature:0,
                     {name:"Hops 3", location:"Adjunct3", temperature:202, step_time:8, drain_time:0},
                     {name:"Hops 4", location:"Adjunct4", temperature:202, step_time:8, drain_time:5},
                     ];
+var default_data_c = [{name:"Preparing To Brew", location:"Prime", temperature:0, step_time:3, drain_time:0},
+                    {name:"Heating", location:"PassThru", temperature:43, step_time:0, drain_time:0},
+                    {name:"Dough In", location:"Mash", temperature:43, step_time:7, drain_time:0},
+                    {name:"Mash 1", location:"Mash", temperature:64, step_time:45, drain_time:0},
+                    {name:"Mash 2", location:"Mash", temperature:68, step_time:0, drain_time:0},
+                    {name:"Mash Out", location:"Mash", temperature:81, step_time:7, drain_time:2},
+                    {name:"Hops 1", location:"Adjunct1", temperature:94, step_time:10, drain_time:0},
+                    {name:"Hops 2", location:"Adjunct2", temperature:94, step_time:8, drain_time:0},
+                    {name:"Hops 3", location:"Adjunct3", temperature:94, step_time:8, drain_time:0},
+                    {name:"Hops 4", location:"Adjunct4", temperature:94, step_time:8, drain_time:5},
+                    ];

```

Comment:
like my earlier comment having all this duplicated for C just makes twice the amount of code to support and maintain as they likely could change due to bug, refactoring, renaming, etc and one might be left out.